marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts

```diff
@@ -176,12 +176,21 @@ export class ProductDetailComponent {
     });
   }
 
+  private isPropertyNullOrAllEmpty(property: DisplayValue | null) {
+    if (property === null) {
+      return true;
+    } else {
```

Comment:
please remove line 182 and 185.

The else here is redundant. if property === null . you already return true

Reply:
Fixed as commented